quaistats/quaistats.go

```diff
@@ -923,3 +964,21 @@ func (s *Service) reportPeers(conn *connWrapper) error {
 	}
 	return conn.WriteJSON(report)
 }
+
+func dirSize(path string) (float64, error) {
+	var size int64
+	err := filepath.WalkDir(path, func(_ string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if !d.IsDir() {
+			fileInfo, err := d.Info()
+			if err != nil {
+				return err
+			}
+			size += fileInfo.Size()
+		}
+		return nil
+	})
+	return float64(size) / 1024 / 1024, err // convert to MB
```

Comment:
For all of these, would it be better to return them as an int at the end? Since it'll be transmitted across the network I'm not sure we want to have super precise decimals. It's up to the consuming application

Reply:
added a second commit with these changes. We can review and decide which implementation we prefer

Reply:
maybe we should still do the MB conversion and just round?
after discussing more with @Djadih

Reply:
spoke with @kiltsonfire 
sticking with float32 for now